app/src/main/java/com/android/feedme/model/viewmodel/CameraViewModel.kt

```diff
@@ -287,10 +286,31 @@ class CameraViewModel : ViewModel() {
    * @param ing The ingredient metadata to update the list with.
    */
   fun updateIngredientList(ing: IngredientMetaData) {
+    if (ing.ingredient.id != "TEST_ID") {
```

Comment:
what is `"TEST_ID"` here ? was it for test purpose ?

Reply:
It's the easiest way to prevent the uses of `IngredientsRepository.instance.getExactFilteredIngredients()` when testing.

Reply:
Ok i see !